pom.xml

```diff
@@ -9,7 +9,7 @@
 
   <groupId>fm.last.commons</groupId>
   <artifactId>lastcommons-test</artifactId>
-  <version>7.0.2-SNAPSHOT</version>
+  <version>7.1.0-SNAPSHOT</version>
```

Comment:
`7.0.2-SNAPSHOT` as the version number is better since this is a bugfix not a new feature.